tests/test_translate.py

```diff
@@ -38,7 +38,7 @@
 import unittest
 import sys
 import os
-sys.path.insert(0, os.path.abspath('..'))
+# sys.path.insert(0, os.path.abspath('..'))
```

Comment:
I can't remember why I made this change. It may have to be reverted.

Reply:
hmm no I think I was wrong. I will revert this change, cause I'm not sure how to run the unit-tests otherwise (the way they are built now) :P